tests/integration-tests/tests/efa/test_efa/test_hit_efa/osu_pt2pt_submit_openmpi.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+BENCHMARK_NAME={{ benchmark_name }}
+OSU_BENCHMARK_VERSION={{ osu_benchmark_version }}
```

Comment:
why not use an arg instead of templating these?

Reply:
Because this script is not executed by the framework but passed to Slurm to be submitted as is.